pages/glossary/main-section-of-content.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Main section of content
+key: main-section-of-content
+unambiguous: true
+objective: false
+---
+
+The _main section of content_ of an [HTML web page][] is its [section of content][] whose content is serving the key purpose of the page and fulfills expectations of users navigating to the page.
```

Comment:
Who defines the key purpose? What if there are multiple important purposes that might be key for different users? Does that mean there are multiple main sections of content?

Reply:
Would we lose anything if we left the key purpose out?
```
The _main section of content_ of an [HTML web page][] is a [section of content][] whose content fulfils the expectations of users navigating to the page.
```